challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ Relative.   It has a slash in front of it so there are more directories in front of it.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ Absolute.  The file exists outside of my computer so it needs to know exactly where it is.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ Relative.  It is known that the file lives inside the project folder.
```

Comment:
We would use a Relative path, but the reason why is because of link breaking. Imagine if we'd zipped up our project and shared it with a friend. When our friend opens the project, the actual locations of all of the files will be different; as a result, any links that used Absolute paths would be broken by the move. Relative paths don't have that problem, because (since the project was all bundled up together) all of the files are in the same positions _relative to each other_.